app/utils/replace-links.js

```diff
@@ -0,0 +1,44 @@
+const legacyExternalLinks = new Set([
+  'https://emberjs.com/api',
+  'https://emberjs.com/blog',
+  'https://emberjs.com/deprecations',
+]);
+
+function isExternalLink(url) {
+  const isExternalLink = !url.startsWith('https://emberjs.com');
+  const isLegacyExternalLink = legacyExternalLinks.has(url);
+
+  return isExternalLink || isLegacyExternalLink;
+}
+
+function replaceInternalLinks(url) {
+  if (isExternalLink(url)) {
+    return url;
+  }
+
+  // Map internal links to the correct route name
+  return url
+    .replace(/^https:\/\/emberjs.com\//, '/')
+    .replace(/^\/builds(\/\w+)$/, '/releases$1')
+    .replace(/\/builds$/, '/releases');
```

Comment:
These lines may be no longer needed so consider removing them.

All [header links](https://github.com/ember-learn/ember-styleguide/blob/ab1d1fc32dd023f287c49d3fd700216ba368771a/addon/constants/links.js) and [footer links](https://github.com/ember-learn/ember-styleguide/blob/7a836bda9ce13e3dbddc166b8f56a0425f8eafe6/addon/constants/es-footer.js) that come from `ember-styleguide` do not include the word `builds` in the URL. We also use `.netlifyredirects` to redirect a user if the URL happens to include `builds`.